src/helpers/schema/schema.helper.ts

```diff
@@ -451,14 +455,15 @@ export class SchemaHelper {
    * @return {{}}
    */
   public static removeProperty(object: {}, key: string): {} {
-    var segments = key.split(".");
-    var currentObject: any = object;
-    var currentPath: string = null;
-    var objectStack: any = [];
+    let i;
+    const segments = key.split(".");
+    let currentObject: any = object;
+    let currentPath: string = null;
+    const objectStack: any = [];
 
-    for (var i = 0; i < segments.length; i++) {
+    for (i = 0; i < segments.length; i++) {
```

Comment:
```suggestion
    for (let i = 0; i < segments.length; i++) {
```
We can remove the declaration of `i` from line 458 so that we can have different block scope.